Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM php:7.0.26-fpm-jessie
```

Comment:
This is an official (as in official from Docker, not official from the PHP core team) image:
- `7.0` because it's the closest version to what we use now
- `fpm` because it's the web-facing version with `php-fpm` (equivalent to `uwsgi` for the Python folks) and not `cli`. I will have to create a separate `cli` image that instead can be used for the long-running process looking at the SQS queue, and for running all tests (that test also the long-running process and do not work in `fpm`)
- `jessie` (Debian oldstable) because there is no `ubuntu` in the official images. The alternative is `alpine` which while being locked down and lean in size, has a different package management system. I tend to be conservative and go for the most similar option.

In any case, we will need a base `elife/php-fpm` (naming to be decided) image so that all PHP projects share the same base.

Reply:
can you have a dual php-fpm/cli base?